doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,87 @@
+
+1. Sometimes people just want to say thank you to the PT team (like in https://github.com/microsoft/PowerToys/issues/7873) and a thread in GitHub Discussions where every one can comment if he just want to say thank you, is much better for the users and the team than open every time a new issue for that.
+
+2. Questions: If people just have questions, it gets difficult by asking them in issues. It's more useful for both sides (team and user) when questions are asked in a different section. Because in GitHub discussion you can mark a reply as the answer and other users can find the answer quicker.
+
+3. Off-topic. Sometimes issue discussions get very and sometimes a little off-topic (like https://github.com/microsoft/PowerToys/issues/7916#issuecomment-724813783) And it would be better if you can open a thread and write "As seen in #XXXX I think the real reason for ... is ..."
+
+4. Tell informations to the community: 
+
+4. More community interaction: More information in the underneath section
+
+### Community interaction
+
+#### Do PowerToys needs this?
+
+Yes because PowerToys is a community project. And PowerToys is a project for power users. And not all power users are programmers (like designers, etc.). And in the moment the community interaction is mostly based on programming.
+
+#### But PowerToys has already community interaction. PowerToys has around 200 new issues in a week.
+
+That's right but these are normally bug reports and feature suggestions and PowerToys needs more things for a strong community. Things like jokes, memes and grand discussion about PowerToys related things.
+
+## Does this hinder the PowerToys team from working on PowerToys?
+
+Not really. One of the core team members needs to set the settings first. But after that the triage team can take care of this section. Most of them don't program for PowerToys so they don't lose time for programming. The programmers of PowerToys don't need to take care of GitHub discussion, because it's only about community stuff where the triage team don't need the help of them.
+
+## Settings
+
+📝General (answers not enabled)
+❓Q&A (answers enabled)
+(💬Off-topic (answers not enabled))
+
+Pinned thread: (conversation locked)
+```
+
+# Hello and welcome to our GitHub discussion section.
+
+Here you can ask questions and say thank you.
+
+
+[Some links to other important threads]
+```
+
+Other threads:
+-	A thread where you can say thank you to the PT team.
+-	A meme thread
+- ...
+
+
+## What when it don't work
+
+If the community denies the feature, it gets too much work, it gets used very low or if it gets filled with spam PowerToys can simply deactivate the feature.
+
+## Definition of success
+
+* In a week more than 12 new threads
+* It don't get too much work for the PT team
+* People discuss with each others about things
+
+## **Summary**
+
+PowerToys should enable GitHub discussions because:
+* It don't take much time away from the developers: The project don't suffer under that.
```

Comment:
> It don't take much time away from the developers: The project don't suffer under that.

I don't think that is a reason

Reply:
Clint said it could possible take too much time from the developers away. But I don't think so. Because of that it is a reason

Reply:
But technically that is not a reason

Reply:
Yes. But I will not remove it

Reply:
In an ideal scenario, having Discussions and not having Discussions will not have an impact on the devs' productivity. That means that it is NOT a reason